spras/btb.py

```diff
@@ -0,0 +1,188 @@
+# need to define a new btb class and contain the following functions
+# - generate_inputs
+# - run
+# - parse_output
+
+import warnings
+from pathlib import Path
+
+import pandas as pd
+
+from spras.containers import prepare_volume, run_container
+# from spras.interactome import (
+#     convert_undirected_to_directed,
+#     reinsert_direction_col_directed,
+# )
+# what type of directionality does btb support?
```

Comment:
It expects a directed graph (it uses networkx's `DiGraph()` object).

Reply:
I think it has been fixed in the newest push?